src/PhpBrew/Command/UseCommand.php

```diff
@@ -26,26 +24,8 @@ public function brief()
         return 'Use php, switch version temporarily';
     }
 
-    public function execute($buildName)
+    public function execute()
     {
-        if (!$buildName) {
-            // This exception is used for tracing tests
-            throw new \Exception('build name is required.');
-        }
-        { // this block is important for tests only
-            $root = Config::getRoot();
-            $home = Config::getHome();
-            if (!file_exists("$root/php/$buildName")) {
-                throw new \Exception("build $buildName doesn't exist.");
-            }
-            putenv("PHPBREW_ROOT=$root");
-            putenv("PHPBREW_HOME=$home");
-            putenv("PHPBREW_PHP=$buildName") or die('putenv failed');
-            putenv("PHPBREW_PATH=$root/php/$buildName/bin");
-            putenv("PHPBREW_BIN=$home/bin");
-        }
-        if (!getenv('TRAVIS')) {
-            $this->logger->warning("You should not see this, if you see this, it means you didn't load the ~/.phpbrew/bashrc script, please check if bashrc is sourced in your shell.");
-        }
+        throw new Exception('The PHP implementation of the "use" command is not supposed to be invoked.');
```

Comment:
maybe you need [`VirtualCommand`](https://github.com/phpbrew/phpbrew/blob/master/src/PhpBrew/Command/VirtualCommand.php)?

Reply:
True. I forgot we have one.